### ISSUE TITLE
cheatsheet

### ISSUE THREAD

**Issue by owocki** [gitcoinco/labs#2]
one thing i think that could come out of your research @Austin Griffith is a document that just lists ELI5 explanations of various things in the ecosystem.

- VDFs
- curved bonding
- gnosis safe
- counter factuals
- metaTx
- plasma

a cheatsheet like that could have a lot of value to this team, to the community

**Comment by Blazingthirdeye** [comment]
I am not sure why this got ignored.  It seems like a great idea.  Maybe I can nudge it ahead.
